Content/.template.config/template.json

```diff
@@ -83,13 +83,21 @@
                 {
                     "choice": "remoting",
                     "description": "add Fable.Remoting (https://zaid-ajaj.github.io/Fable.Remoting/) to server and client"
+                },
+                {
+                    "choice": "bridge",

```

Comment:
lets add it to Tests as in previous PR

Reply:
Sorry, sent it on a limited internet connection and has kinda of hurried.